pulp_smash/tests/pulp3/file/utils.py

```diff
@@ -1,9 +1,58 @@
 # coding=utf-8
 """Utilities for tests for the file plugin."""
-from pulp_smash.tests.pulp3 import utils
+from urllib.parse import urljoin
+
+from pulp_smash import api, utils
+from pulp_smash.constants import FILE_FEED_URL
+from pulp_smash.tests.pulp3.constants import (
+    FILE_CONTENT_PATH,
+    FILE_REMOTE_PATH,
+    REPO_PATH,
+)
+from pulp_smash.tests.pulp3.pulpcore.utils import gen_repo
+from pulp_smash.tests.pulp3.utils import (
+    require_pulp_3,
+    require_pulp_plugins,
+    sync,
+)
+
+
+def gen_remote(url):
```

Comment:
As we talked, please, move the gen_remote to `pulp3/utils.py` since it is not plugin dependent.